Solutions/LastPass/Analytic Rules/HighlySensitivePasswordAccessed.yaml

```diff
@@ -23,7 +23,7 @@ relevantTechniques:
  - T1087
 query: |
   let watchlist = (_GetWatchlist("LastPass") | project name);
-  LastPass_BYOC_CL
+  LastPassNativePoller_CL
   | where Data_s in (watchlist)
   | extend timestamp = , AccountCustomEntity = Username_s, IPCustomEntity = IP_Address_s, URLCustomEntity = Data_s
```

Comment:
Hey @ThijsLecomte , there is error in query for **timestamp= ,** as it is blank, please check and update, thanks.